src/components/ActiveDot/activeDot.ts

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import colors from "../../core/constants/colors";
+
+const ActiveDotWrapper = styled.span<{ $isActive: boolean }>`
+  background: ${(props) =>
+    props.$isActive ? colors.GREEN_COLOR : colors.RED_COLOR};
+  border-radius: 50%;
+  margin: 10px;
+  height: 10px;
+  width: 10px;
+  transform:  scale(1);
+  transition: transform 0.2s;
+  z-index: 0;
```

Comment:
why this 0 value needed

Reply:
Resolved in b4551ebb492936dce5682ce1c6defe8feaa5b196